crates/bevy_asset/src/lib.rs

```diff
@@ -258,6 +258,30 @@ pub struct AssetPlugin {
     pub mode: AssetMode,
     /// How/If asset meta files should be checked.
     pub meta_check: AssetMetaCheck,
+
+    pub out_of_bounds_mode: OutOfBoundsMode,
+}
+
+/// Determines how asset paths that go out of bounds are handled.
+///
+/// An out of bounds asset path is one that attempts to load a file that
+/// is not inside the default asset folder or another asset source. This usually
+/// means it is either an absolute address or it contains enough "../"s to
+/// exit the designated asset folder.
+///
+/// The bevy team strongly discourages using [`Allow`](OutOfBoundsMode::Allow) if your
```

Comment:
It feels weird to refer to "the bevy team", you can just say that it's discouraged.